### ISSUE TITLE
Master React, Quick Code tutorial. Part 3

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/react-only-intern-20#4]
https://medium.com/quick-code/lets-learn-react-chapter-3-components-components-components-3492f771d623

## Expected result

- [ ] Read this tutorial chapter
- [ ] follow code steps
- [ ] move your code to this repository(at least one commit per day)
- [ ] explain to me, what did you learn from it in a few sentences(not less than 5)
- [ ] pass the test section
- [ ] after reviewing your progress and closing this issue - you can move to the next chapter.

## Test
- [ ] similar component for day checker is month checker. Create it.
- [ ] btw, can you have a some sort of component between month and day - week, right?
with ability(method) - to grab only work days, and other method for weekends?
- [ ] separate some of repeating functionality from that HTML code too...
- [ ] I want to have one array - that will contain days and month together. so when you'll need to render a 'calendar' table - you'll use 2 forEach methods.
- [ ] and i want to have all static data - like titles, image url, content - into one file, make it like array. sample: https://github.com/ChickenKyiv/recipe-search-react/tree/master/src/data
#1

**Comment by dscace** [comment]
I learned that you should 'import' all of your different components at the top of your App.js.

React components are reusable throughout the app.

When creating a new component class, remember to always include an export statement at the bottom.

You'll get an error if you tell to render to 'return' more than 1 element.

To 'return' multiple elements, you need to either wrap them in a 'div' or make it an array.

**Comment by atherdon** [comment]
tell me when you'll finish test section - i'll review your code

**Comment by dscace** [comment]
Still working on it. I learned that I can only close issues in comments when I commit to the default branch. Can we make 'checkers' the default branch?

**Comment by atherdon** [comment]
no, i like the way how i can review your progress by viewing pull requests


On 26-Jun-2018 at 07:36 AM, Derek Scace wrote:


Still working on it. I learned that I can only close issues in comments
when I commit to the default branch. Can we make 'checkers' the default
branch?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/react-only-intern-20/issues/4#issuecomment-400209826>,
or mute the thread

**Comment by atherdon** [comment]
@dscace can you tell me - did you complete the last to tasks from test section?

**Comment by dscace** [comment]
Hey Arthur! No problem with you being busy. I’m without my computer for a couple days. I’m going to tackle the rest of the test and then move to the next section Friday night and work each evening this weekend as well. I’ll send you a message Friday night! Thanks

**Comment by dscace** [comment]
@atherdon I hope that’s ok! I’m excited to be working on this. I just need to be more active with it.

**Comment by atherdon** [comment]
sure - thank oyu for letting me know. take your time


On 05-Jul-2018 at 04:13 AM, Derek Scace wrote:


Hey Arthur! No problem with you being busy. I’m without my computer for a
couple days. I’m going to tackle the rest of the test and then move to the
next section Friday night and work each evening this weekend as well. I’ll
send you a message Friday night! Thanks

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/react-only-intern-20/issues/4#issuecomment-402601686>,
or mute the thread

**Comment by atherdon** [comment]
curious to see your progress, can you create a pull request with latest changes? I'll merge it to main branch, so we'll have a right git-flow workflow

**Comment by dscace** [comment]
Was away this weekend playing a festival with my band. Trying to figure out the day/month array right now.

**Comment by dscace** [comment]
Not many videos have been posted yet. He's a short one: https://youtu.be/-7Theo0ngs8

You want days and months in one array? Been trying to figure that out around google.. I'll keep at it tonight.
